layouts/index.html

```diff
@@ -138,90 +111,20 @@ <h2 class="billboard-second_text">Focus on developing your applications. We mana
-        <h3>Security</h3>
-        <p>
-          cloud.gov is deployed on a hardened operating system. The platform and every application
-          on it are centrally scanned and audited. Because the platform and its controls are consistent,
-          every security improvement to cloud.gov in turn is a security improvement for your products.
-        </p>
-      </div>
-      <div class="four_up-part">
-        <div class="icon-container">
-          <svg class="icon icon-alt">
-            <use xlink:href="assets/img/cloudgov-sprite.svg#i-user_researched"/>
-          </svg>
-        </div>
-        <h3>Conformity with federal rules</h3>
-        <p>
-          Every NIST control for the platform, infrastructure, and human layers of cloud.gov is
-          documented and ready to use in many different contexts. The documentation is updated
-          automatically with every update to the platform, and it’s yours to use to shave dozens
-          of hours off the usual app-by-app, copy-and-paste compliance process.
-        </p>
-      </div>
-    </div>
-  </div>
-  <!--
-  <i class="hex_icon hex_icon-primary section-icon"
-      style="height: 5.625rem; width: 4.9rem">
-    <svg class="icon">
-      <use xlink:href="assets/img/cloudgov-sprite.svg#i-checked"/>
-    </svg>
-  </i>
-  -->
-</section>
-
-<section class="usa-content section-middle section-light">
+<section class="usa-content section section-dark section-chevron-alt">
   <div class="usa-grid">
     <div class="section-content usa-content">
-      <h1>It’s a solution that keeps getting better.</h1>
-      <p>
-        We built cloud.gov on the open source, public Cloud Foundry project. We completed the first prototype in about three months, and we started launching production apps about a month after that. We continue to fine-tune it based on the needs of the apps that teams have built.
-      </p>
-      <p>
-        Because it’s open source, we can continue to take advantage of improvements to the operability and security of Cloud Foundry. It helps us scale.
-      </p>
-      <p>
-        Because it’s an agile project, cloud.gov is always under development. As we observe its use, we will continue to add new capabilities. Therefore, we do not say it is “complete” and we probably never will.
-      </p>
+      <h1>Are you right for cloud.gov?</h1>
+      <p>cloud.gov was built by a government team for government teams. Here’s who we’re looking
+        for:</p>
+        <ul>
+          <li>You’re in a federal government organization that employs teams — either in-house
+            or on contract — with the power to push applications.</li>
+          <li>You or your teams can can use the IAA/MOU process to pay GSA for cloud.gov.</li>
+          <li>Your applications are FISMA-Moderate or lower, and the engineering approach values
+              clarity (for example, the application listens to a single port and stores
+              configuration in the environment).</li>
```

Comment:
These sound like two completely separate ideas. Shouldn't they be their own line items?